package.json

```diff
@@ -16,7 +16,10 @@
   ],
   "main": "src/index.tsx",
   "dependencies": {
-    "@solana/web3.js": "^1.47.1"
+    "@solana/web3.js": "1.63.1",
+    "react": "^18.2.0",
```

Comment:
I'm afraid this will break CodeSandbox - we found out the hard way that i has a hard limit on its number of dependencies and web3.js takes up a lot of that. Sant's solution was to move these to peer and dev dependencies. Can you restructure and test to see if this fork will work with CodeSandbox?

See previous solutions: 
- https://github.com/phantom-labs/sandbox/pull/22
- https://github.com/phantom-labs/sandbox/pull/23